apps/trading/components/card/card.tsx

```diff
@@ -86,11 +89,16 @@ export const CardTable = (props: HTMLProps<HTMLTableElement>) => {
   );
 };
 
-export const CardTableTH = (props: HTMLProps<HTMLTableHeaderCellElement>) => {
+interface CardTableTHProps extends HTMLProps<HTMLTableHeaderCellElement> {
```

Comment:
I don't think you need to extends the props with testId: string. data-testid is a valid normal HTMLProp so the component should accept just passing data-testid as a prop directly

Reply:
Fixed